src/main/java/no/nav/familie/inntektsmelding/integrasjoner/joark/JoarkTjeneste.java

```diff
@@ -0,0 +1,133 @@
+        this.organisasjonTjeneste = organisasjonTjeneste;
+        this.personTjeneste = personTjeneste;
+    }
+
+
+    public String journalførInntektsmelding(String XMLAvInntektsmelding, InntektsmeldingEntitet inntektsmelding) {
+        var request = opprettRequest(XMLAvInntektsmelding, inntektsmelding);
+        try {
+            var response = joarkKlient.opprettJournalpost(request, false);
+            // Kan nok fjerne loggingen etter en periode i dev, mest for feilsøking i starten.
+            LOG.info("Journalført inntektsmelding fikk journalpostId " + response.journalpostId());
+            LOG.info("Ble journalført inntektsmelding ferdigstilt:  " + response.journalpostferdigstilt());
+            return response.journalpostId();
+        } catch (Exception e) {
+            throw new IllegalStateException("Klarte ikke journalføre innteketsmelding " + e);
+        }
+    }
+
+    private OpprettJournalpostRequest opprettRequest(String xmlAvInntektsmelding, InntektsmeldingEntitet inntektsmeldingEntitet) {
+        boolean erBedrift = inntektsmeldingEntitet.getArbeidsgiverIdent().length() == 9;
+        AvsenderMottaker avsenderMottaker = erBedrift ? lagAvsenderBedrift(inntektsmeldingEntitet) : lagAvsenderPrivatperson(inntektsmeldingEntitet);
+        var request = OpprettJournalpostRequest.nyInngående()
+            .medTittel(JOURNALFØRING_TITTEL)
+            .medAvsenderMottaker(avsenderMottaker)
+            .medBruker(lagBruker(inntektsmeldingEntitet.getAktørId()))
+            .medBehandlingstema(mapBehandlingTema(inntektsmeldingEntitet.getYtelsetype()))
+            .medDatoMottatt(inntektsmeldingEntitet.getOpprettetTidspunkt().toLocalDate())
+            .medTema(mapTema(inntektsmeldingEntitet.getYtelsetype()))
+            .medEksternReferanseId(UUID.randomUUID().toString())
+            .medJournalfoerendeEnhet(JOURNALFØRENDE_ENHET)
+            .medKanal(KANAL)
+            .medDokumenter(lagDokumenter(xmlAvInntektsmelding))
+            .build();
+        return request;
+    }
+
+    private List<DokumentInfoOpprett> lagDokumenter(String xmlAvInntektsmelding) {
+        var dokumentXML = new Dokumentvariant(Dokumentvariant.Variantformat.ORIGINAL, Dokumentvariant.Filtype.XML,
+            xmlAvInntektsmelding.getBytes(StandardCharsets.UTF_8));
+
+        var dokumentPDF = new Dokumentvariant(Dokumentvariant.Variantformat.ARKIV, Dokumentvariant.Filtype.PDF,
+            "PDF_HER".getBytes(StandardCharsets.UTF_8)); // TODO Her må vi sette inn PDF
+
+        var builder = DokumentInfoOpprett.builder()
+            .medTittel(JOURNALFØRING_TITTEL)
+            .medBrevkode(BREVKODE_IM)
+            .leggTilDokumentvariant(dokumentXML)
+            .leggTilDokumentvariant(dokumentPDF);
+
+        return Collections.singletonList(builder.build());
+    }
+
+    private String mapTema(Ytelsetype ytelsetype) {
+        return switch (ytelsetype) {
+            case FORELDREPENGER, SVANGERSKAPSPENGER -> "FORS_SVA";
```

Comment:
Noe usikker på den - ser at utgående brev får Tema "FOR".

Reply:
Uansett om det er FP eller SVP

Reply:
Bra catchet, endrer til "FOR"!